mkposters/mkposter.py

```diff
@@ -42,8 +70,15 @@ def mkposter(datadir):
         html_out = rf"""<!doctype html>
         <html>
         <head>
+        <body style="background-color:{background_color}">
```

Comment:
This isn't well-formed HTML: you've added an open body tag in the middle of the head.

Personally I subscribe to the "do all styling through CSS" approach, and to never use inline styles.

Perhaps the best thing to do here would be to allow for user-specified SCSS (alongside the markdown). And then let the background color just be set there if desired.

Reply:
Good point on user specified SCSS. I started editing the `custom.scss` first and wanted to generalize at minimum the background color so added some non-ideal html. I agree adding a custom option for a user file is the right approach. I can work on that in this PR.

Reply:
Addressed in most recent push as `scss` optional user specified path.